package/src/values/__tests__/selector.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import type { AnimatedProp } from "../../renderer/components";
+import { Fill } from "../../renderer/components";
+import { mountCanvas } from "../../renderer/__tests__/setup";
+import { processResult } from "../../__tests__/setup";
+import { Selector } from "../selector";
+
+interface ArrayProps {
+  color: AnimatedProp<string>;
+}
+
+const TestSelector = ({ color }: ArrayProps) => {
+  return <Fill color={color} />;
+};
+
+describe("Value Selector", () => {
```

Comment:
It may be over-cautious; could we add tests for `isValue` and `isSelector`? Mainly checking that `isValue(selector)` and `isSelector(value)` will always return false?